Makefile

```diff
@@ -1,17 +1,30 @@
 DOCKER_COMPOSE = docker-compose -f docker-compose.yml
 BUNDLE_FLAGS=
+
 ifdef DEPLOYMENT
   BUNDLE_FLAGS = --without test development
 endif
 
 ifndef JENKINS_URL
```

Comment:
Should we just remove all references to Jenkins?
Are we keeping this backwards compatible as we make the switch to concourse?

Reply:
leaving them in to make the switch easier